parsing/parsetree.mli

```diff
@@ -22,7 +22,7 @@ type constant =
   (* 3 3l 3L 3n
 
      Suffixes [g-z][G-Z] are accepted by the parser.
-     Suffixes except 'l', 'L' and 'n' are rejected by the typechecker
+     Suffixes except 'l', 'L' and 'n' are rejected by the type-checker
```

Comment:
(In OCaml trunk) actual usages of :  

- ' typechecker'  : 27
- ' type checker'  : 103
- ' type-checker'  : 378  
  
There is a big winner, I think.